plugins/search/src/components/SearchResultPager/SearchResultPager.tsx

```diff
@@ -29,6 +29,10 @@ const useStyles = makeStyles(theme => ({
   },
 }));
 
+/**
+ * @public
+ * @deprecated Moved to `@backstage/plugin-search-react`.
```

Comment:
What about having a more instructional focused message here so that the user who is using the component can take action? I guess same goes for other similar deprecation messages.

```suggestion
 * @deprecated Import from `@backstage/plugin-search-react` instead.
```